virtual_rainforest/models/abiotic_simple/abiotic_simple_model.py

```diff
@@ -160,25 +158,25 @@ def from_config(
     def setup(self) -> None:
         """Function to set up the abiotic simple model."""
 
-        # setup_variables = simple_regression.setup_simple_regression(
-        #     layer_roles=self.layer_roles,
-        #     data=self.data,
-        #     initial_soil_moisture=self.initial_soil_moisture,
-        # )
-        # update_data_object(data=self.data, output_dict=setup_variables)
+        setup_variables = simple_regression.setup_simple_regression(
+            layer_roles=self.layer_roles,
+            data=self.data,
+            initial_soil_moisture=self.initial_soil_moisture,
+        )
```

Comment:
I am not sure I fully understand what you mean here.

First, `super().__init_()` is called in the `__init__` and I tested that all required variables are loaded. Did I miss something here?

Second, the variables that are created in the `setup_simple_regression` are not inputs so I don't see how they would be coming from the initial data. Also, they might be different depending on the `abiotic` model selected, so I think their initialization it should stay within the model somewhere. If the setup function is not a good place, what would be an alternative? Maybe config?

Lastly, I see how the inputs that come from the `plants` should be in `conftest` for testing, which in fact they are (jn the `dummy_climate_data` fixture.). However, in order for us to try `vr_run` we will need them available in `data`. An alternative could be to have a `plants` dummy that just created these variables so we can move on. Would you agree?

Reply:
Phew, that happens to me all the time with the code that is folded away, so all good :-) 

About initializing nan DataArrays and updating the data object, I think this might have party evolved from the class structure and the initialization of class attributes, and partly from the need to avoid updating the data object directly. Looking now at how I implemented updating the data object, it is not necessary for a variable to be there in the first place, so I can drop all variables from setup that don't contain values.

For the `plants` variables, I think a NetCDF file is a very good solution. I'm happy to create that 👍

Reply:
I removed the `setup_regression` function and added the initialization of soil moisture and vpd reference to the `abiotic_simple_model`.